tests/main_tests/median_test.py

```diff
@@ -0,0 +1,16 @@
+"""
+This script tests the median function.
+"""
+import pandas as pd
+
+from src.utils.helper_functions.calculations import find_median
+
+
+def test_find_median():
+    df = pd.DataFrame({"feature": [1, 3, 5, 7, 9]})
+    feature_name = "feature"
+    expected_median = 5.0
+    expected_index = 2
+    median, median_index = find_median(df, feature_name)
```

Comment:
Consider adding some comments here to help describe what's being done.

Reply:
Also good point, adding accordingly